setup.py

```diff
@@ -76,24 +76,29 @@
 import os
 import re
 import shutil
-from distutils.core import Command
+import textwrap
 from pathlib import Path
 
-from setuptools import find_packages, setup
+from setuptools import find_packages, setup, Command
 
 
 # Remove stale transformers.egg-info directory to avoid https://github.com/pypa/pip/issues/5466
 stale_egg_info = Path(__file__).parent / "transformers.egg-info"
 if stale_egg_info.exists():
     print(
-        (
-            "Warning: {} exists.\n\n"
-            "If you recently updated transformers to 3.0 or later, this is expected,\n"
-            "but it may prevent transformers from installing in editable mode.\n\n"
-            "This directory is automatically generated by Python's packaging tools.\n"
-            "I will remove it now.\n\n"
-            "See https://github.com/pypa/pip/issues/5466 for details.\n"
-        ).format(stale_egg_info)
+        textwrap.dedent(
+            f"""
+            Warning: {stale_egg_info} exists.
+
+            If you recently updated transformers to 3.0 or later, this is expected,
+            but it may prevent transformers from installing in editable mode.
+
+            This directory is automatically generated by Python's packaging tools.
+            I will remove it now.
+
+            See https://github.com/pypa/pip/issues/5466 for details.
+            """
```

Comment:
We can just leave the f-string. The likelihood of this warning popping is next to 0 now that version 2.xxx are so old.

Reply:
I do not change the string output but refactored it with `textwrap` without `\n`s. I have reverted it if is not relevant to this PR.